docs/content/contributing/bicep/bicep-contribution-flow/_index.md

```diff
@@ -184,6 +184,14 @@ It is highly recommended to start leveraging Option 1 below to adopt OpenID Conn
 
 {{% expand title="➕ Option 1 [Recommended]: OIDC - Configure a federated identity credential" %}}
 
+{{% notice style="warning" title="Known issue: Service Principal vs User-Assigned / Managed Identity" %}}
+
+While it is possible to set federated credentials up for both a Service Principal and a Managed Identity, it is highly recommended to use the later.
```

Comment:
I would not even put that as a recommendation as it is precisely mentioned in the step by step guidelines to use that. What we can do is to add a note on the reason for using uai vs using sp

Reply:
Also I'd suggest to provide reference to the official doc mentioning this (e.g. https://github.com/marketplace/actions/azure-login#login-with-openid-connect-oidc-recommended or anything better/less confusing)

Reply:
Ok let me rephrase it a bit. But would you do away with the warning itself and just write it in the text? I'd recommend to keep it as a 'eye-catching' statement as apparently, its otherwise missed.

Reply:
Updated @eriqua. 💪

Reply:
The format looks good and agree on the pop-up note. I added a suggestion for the wording, resolving this one